<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralPage</class>
 <widget class="QWidget" name="GeneralPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
	width:auto;
	height:30px;
	border-radius: 5px;
	border: 1px solid black ;
}

QCheckBox::indicator{
	height: 50px;
	width: 50px;
}

QCheckBox::indicator::unchecked{
	image: url(:/icons/imgs/chk_off.png)
}

QCheckBox::indicator::checked{
	image: url(:/icons/imgs/chk_on.png)
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Smiley Sans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>General Options</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="dohUrl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Smiley Sans</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="placeholderText">
      <string>DoH Server Address</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="enV6">
     <property name="font">
      <font>
       <family>Smiley Sans</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Enable IPv6</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="allowLan">
     <property name="font">
      <font>
       <family>Smiley Sans</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Allow LAN</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="enV4">
     <property name="font">
      <font>
       <family>Smiley Sans</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Enable IPv4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="dnsAddr">
     <property name="font">
      <font>
       <family>Smiley Sans</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="placeholderText">
      <string>Backup DNS Address</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
